### ISSUE TITLE
oneOf with message needs linefeed before first message

### ISSUE THREAD

**Issue by ryanskinner** [asyncapi/markdown-template#5]
#### Describe the bug
When using the `oneOf` function under `message`, the markdown (MD) document shows the first message line `##### Message #1` on the same line as `Accepts **one of** the following messages:`

#### How to Reproduce
Use the following yml and export to MD.  The playground.asyncapi.io can also be used.
```yml
asyncapi: '2.0.0'
defaultContentType: application/json
info:
  title: Example OneOf Issue
  version: '1.0.0'


components:
  messages:
    successMessage:
      name: Success
      payload:
          type: object
          properties:
            result:
              type: string
              examples:
                - success

    failureMessage:
      name: Failure
      payload:
          type: object
          properties:
            error:
              type: object
              properties:
                errorCode:
                  type: integer              
                errorMessage:
                  type: string
          examples:
            - 
              error:
                errorCode: 404
                errorMessage: Something messed up

channels:
  some.channel:
    subscribe:
      description: this doesn't show in markdown!
      message:
        oneOf:
          - $ref: '#/components/messages/successMessage'
          - $ref: '#/components/messages/failureMessage'
```

#### Expected behavior
I believe the `##### Message #1` should start on the next line, so that they header can show properly.

**Comment by derberg** [comment]
@ryanskinner there is a PR you can check out

**Comment by ryanskinner** [comment]
@derberg confirmed the fix in 0.3.1.
Thanks for the quick fix.
